source/administration/minio-console.rst

```diff
@@ -170,6 +170,39 @@ Each bucket has :guilabel:`Manage` and :guilabel:`Browse` buttons.
 - Select :guilabel:`Browse` to view the contents of the bucket. 
   You can view and download individual objects, upload new objects, or use the :guilabel:`Rewind` function to view only those :ref:`versions <minio-bucket-versioning>` of an object which existed at the selected timestamp.
 
+.. _minio-console-user-access-keys:
+
+Access Keys
+-----------
+
+.. image:: /images/minio-console/console-access-keys.png
+   :width: 600px
+   :alt: MinIO Console Access Keys
+   :align: center
+
+The :guilabel:`Access Keys` section displays all :ref:`minio-idp-service-account` associated to the authenticated user. 
+
+Access Keys support providing applications authentication credentials which inherit permissions from the "parent" user.
+
+For deployments using an external identity manager such as Active Directory or an OIDC-compatible provider, access keys provide a way for users to create long-lived credentials.
+
+- You can select the access key row to view its custom policy, if one exists.
+
+   You can create or modify the policy from this screen.
+   Access key policies cannot exceed the permissions granted to the parent user.
+
+- You can create a new access key by selecting the :guilabel:`Create service account` button.
+
+   The Console auto-generates an access key and password for the account.
+   You can override these values as necessary.
+
+   You can set a custom policy for the service account that further restricts the permissions granted to the account.
```

Comment:
Might be worth doing a search for `service account` and `account`. They both need to be switched here for `access key`.